.github/CONTRIBUTING.md

```diff
@@ -24,7 +24,46 @@ Also, try to include **steps to reproduce** the issue and expand on the **actual
 
 If you would like to contribute to Parity Ethereum, please **fork it**, fix bugs or implement features, and [propose a pull request](https://github.com/paritytech/parity-ethereum/compare).
 
-Please, refer to the [Coding Guide](https://wiki.parity.io/Coding-guide) in our wiki for more details about hacking on Parity.
+### Labels & Milestones
+
+We use [labels](https://github.com/paritytech/parity-ethereum/labels) to manage PRs and issues and communicate state of a PR. Please familiarize yourself with them. Furthermore we are organizing issues in [milestones](https://github.com/paritytech/parity-ethereum/milestones). Best way to get started is to a pick a ticket from the current milestone tagged [`easy`](https://github.com/paritytech/parity-ethereum/labels/Q2-easy%20%F0%9F%92%83) and get going or [`mentor`](https://github.com/paritytech/parity-ethereum/labels/Q1-mentor%20%F0%9F%95%BA) and get in contact with the mentor offering their support on that larger task.
+
+### Rules
+
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+
+* **No `--force` pushes** or modifying the master branch history in any way. If you need to rebase, ensure you do it in your own repo.
+* **Non-master branches**, prefixed with a short name moniker (e.g. `gav-my-feature`) must be used for ongoing work, and include the associated issue ID (if any) in the branch name.
+* **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+* A pull-request *must not be merged until CI* has finished successfully.
+* Contributors should adhere to the [Parity Ethereum Style Guide](https://wiki.parity.io/Parity-Ethereum-Style-Guide).
+
+### Merge Process
```

Comment:
I have concerns with this whole paragraph. The process it describes is not the one we use. No mention of the two-reviewers requirement (even for insubstantial PRs currently), and perhaps particularly concerning: it seems like it allows merging with two positive reviews even though there's a third reviewer that is strongly opposed. I don't think we want to end up in a situation in which we are forced to merge something some of us do not agree with. Such cases are rare (ever happened?) but if/when they do, forcing a merge by appeal to these 48-hour rules etc seems like asking for trouble. Please reconsider this.

Reply:
I've addressed this in commit b82096038912347d5c673b5c6f50fc5b8be741ce

Reply:
I've addressed this in commit b82096038912347d5c673b5c6f50fc5b8be741ce